assets/js/romo/picker.js

```diff
@@ -50,7 +51,18 @@ RomoPicker.prototype.doRefreshUI = function() {
 }
 
 RomoPicker.prototype.doSetValue = function(value) {
-  this.romoOptionListDropdown.doSetNewValue(value);
+  $.ajax({
+    type:    'GET',
+    url:     this.elem.data('romo-picker-url'),
+    data:    { 'value': value },
+    success: $.proxy(function(data, status, xhr) {
+      this.doSetSelectedValueAndText(data[0].value, data[0].displayText);
```

Comment:
@kellyredding - You might need to handle if `data` is empty here. It's possible we could set an invalid value and in that case it should probably not show anything selected?